### PR TITLE
Web API  Improvements

### PR DESCRIPTION
**Pull request by mrlacey**
For #3180 & #3181

- Renamed from 'Mobile App Service'
- Now using SampleDataService & SampleOrder
- Removed EntityFrameworkCore reference
- Restructured templates to allow for Prism variations
- Added `TODO WTS` comments
- Updated route definition

### CODE REVIEW

**Comment by sibille** [comment]
@mrlacey, can we use the Metapackage Microsoft.AspNetCore.App? I don't think the postaction supports metapackages, but think we can add it on the csproj file directly as vs does when you create a asp.net core web api.

<PackageReference Include="Microsoft.AspNetCore.App" />
(https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-2.1)

**Comment by mrlacey** [comment]
> @mrlacey, can we use the Metapackage Microsoft.AspNetCore.App? I don't think the postaction supports metapackages, but think we can add it on the csproj file directly as vs does when you create a asp.net core web api.
> 
> (https://docs.microsoft.com/en-us/aspnet/core/fundamentals/metapackage-app?view=aspnetcore-2.1)

This is included.
screenshot from app I just generated.
image

Do you mean to update to the 2.2.5 version?

**Comment by sibille** [comment]
No, I'm saying have `<PackageReference Include="Microsoft.AspNetCore.App" />` on the csproj instead of `<PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.11"/>`.

File: templates/Uwp/Services/SqlServerDataService.Prism/.template.config/template.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "wts.platform": "Uwp",
     "wts.group": "Data",
     "wts.version": "1.0.0",
-    "wts.displayOrder": "1",
+    "wts.displayOrder": "s",
```
Comment (sibille) [inline templates/Uwp/Services/SqlServerDataService.Prism/.template.config/template.json]: Should this be a 2?

Comment (mrlacey) [reply templates/Uwp/Services/SqlServerDataService.Prism/.template.config/template.json]: Yes. Will also update the validator tests as they should have picked that up.

Comment (mrlacey) [reply templates/Uwp/Services/SqlServerDataService.Prism/.template.config/template.json]: There is a test that should have caught this. Now to investigate why it didn't...

**Comment by sibille** [comment]
Bringing in those changes, we can adapt the package reference later

**Pull request merged**